### PR TITLE
Added SpatialFeatures.unsupported_geojson_options.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: django/contrib/gis/db/backends/base/features.py [hunk 1]
```diff
@@ -51,6 +51,9 @@ class BaseSpatialFeatures:
     # Can SchemaEditor alter geometry fields?
     can_alter_geometry_field = True
 
+    # Does AsGeoJSON() support the crs option?
+    supports_geojson_crs = True
```
Comment (felixxm) [inline django/contrib/gis/db/backends/base/features.py:55]: We have two more options that are not supported on Oracle, co maybe we should add a set with supported options and handle all of cases in a single feature, e.g.
```python
supported_geojson_options = {'bbox', 'crs', 'precision'}
```

Comment (timgraham) [reply django/contrib/gis/db/backends/base/features.py:55]: That seems good. My only concern is that it's unclear if the flags that apply to a single database are truly generalizable. For example, there's also:
https://github.com/django/django/blob/4cd77f97a2d30d58841041a73d277ec4bfb7b530/tests/gis_tests/geoapp/test_functions.py#L91-L92
I feel like we'll have to continue some vendor checks. My thinking was to replace some usage of `gis_tests.utils.<vendor>` to `connection.ops.<vendor>`.

Comment (felixxm) [reply django/contrib/gis/db/backends/base/features.py:55]: > That seems good. My only concern is that it's unclear if the flags that apply to a single database are truly generalizable. For example, there's also:

Yes, but we have at least a few less vendor checks, and it seems better than three feature flags `supports_geojson_(crs/bbox/precision)`.

> I feel like we'll have to continue some vendor checks.

Unfortunately, yes.

**Comment by timgraham** [comment]
buildbot, test on oracle.

**Comment by felixxm** [comment]
Thanks

**Pull request merged**